### PR TITLE
Added the project page

### PR DESCRIPTION
**Pull request by idacarlin95**

### CODE REVIEW

File: styles/projectstyle.css [hunk 1]
```diff
@@ -0,0 +1,81 @@
+* {
+    box-sizing: border-box;
+}
+
+html {
+    height: 100vh;
+}
+
```
Comment (Alexander-Lindstroem) [inline styles/projectstyle.css:8]: This code already exists in the main .css so you can remove it

File: styles/projectstyle.css [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+.header {
+    background-color: #C1DFF0;
+    padding: 10px;
+}
+
+.header__page-title {
+    color: #2D848A;
+    text-align: center;
+    font-size: 4rem;
+}
```
Comment (Alexander-Lindstroem) [inline styles/projectstyle.css:24]: Same as above

File: styles/projectstyle.css [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+html {
+    height: 100vh;
+}
+
+body {
+    background-color: #88CCF1;
+    display: flex;
+    margin: 0;
```
Comment (Alexander-Lindstroem) [inline styles/projectstyle.css:12]: Same as above

File: styles/projectstyle.css (hunk 1 above)
Comment (idacarlin95) [reply styles/projectstyle.css:8]: Problem fixed!

**Pull request merged**